app/src/main/java/com/gathering/android/event/home/SortDialogFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.gathering.android.event.home
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import com.gathering.android.R
+import com.gathering.android.databinding.FrgSortDialogBinding
+
+
+class SortDialogFragment(private val eventListViewModel: EventListViewModel) : DialogFragment() {
```

Comment:
feel free to do this after you addressed all the other comments or create another refactor ticket for it to be done later:

currently none of Filter or Sort dialogs keep the state of current sort or filter. we can send the current sort to this class via constructor and set the previously set values accordingly.

we already have the previously set filter and sort values in view model. it should be straight forward to take those values here.

Reply:
i will create a separate ticket for this.

Reply:
add a TODO with ticket number to top of both classes please. thanks